mlir/test/Conversion/ArithToEmitC/arith-to-emitc.mlir

```diff
@@ -48,6 +48,16 @@ func.func @arith_integer_ops(%arg0: i32, %arg1: i32) {
   %3 = arith.divsi %arg0, %arg1 : i32
   // CHECK: emitc.rem %arg0, %arg1 : (i32, i32) -> i32
   %4 = arith.remsi %arg0, %arg1 : i32
+  // CHECK: emitc.bitwise_and %arg0, %arg1 : (i32, i32) -> i32
+  %5 = arith.andi %arg0, %arg1 : i32
+  // CHECK: emitc.bitwise_or %arg0, %arg1 : (i32, i32) -> i32
+  %6 = arith.ori %arg0, %arg1 : i32
+  // CHECK: emitc.bitwise_xor %arg0, %arg1 : (i32, i32) -> i32
+  %7 = arith.xori %arg0, %arg1 : i32
+  // CHECK: emitc.bitwise_left_shift %arg0, %arg1 : (i32, i32) -> i32
+  %8 = arith.shli %arg0, %arg1 : i32
+  // CHECK: emitc.bitwise_right_shift %arg0, %arg1 : (i32, i32) -> i32
+  %9 = arith.shrsi %arg0, %arg1 : i32
```

Comment:
While you are at it, maybe also support `shrui`?

Reply:
`shrui` requires a conversion to unsigned. I didn't include operations that needed this (and, well, turns out I missed on some).